### ISSUE TITLE
[Feature Request] Delete button in notification

### ISSUE THREAD

**Issue by rockstarrz** [w2sv/FileNavigator#90]
### Overview

Sometimes i need to delete a newly downloaded file so to do that i have to go to file manager and then delete. A quick delete button would be very useful

**Comment by w2sv** [comment]
Yes I'll add it. I'll close this issue though since it's a duplicate of #79.

**Comment by w2sv** [comment]
I added the feature in https://github.com/w2sv/FileNavigator/releases/tag/0.2.4 :)
